packages/core/src/store/level-state-store.ts

```diff
@@ -22,8 +24,11 @@ export class LevelStateStore implements StateStore {
     /**
      * Open pinning service
      */
-    async open(): Promise<void> {
-        this.#store = new Level(this.storePath);
+    async open(networkName: string): Promise<void> {
+        // Always store the pinning state in a network-specific directory
+        const storePath = path.join(this.storeRoot, networkName)
+        await fs.mkdir(storePath, { recursive: true }) // create dir if it doesn't exist
```

Comment:
Can we add some error handling here?

For example if the process doesn't have write access to the given path, it'd be helpful to return a descriptive error and im guessing we want to exit in that case.

Reply:
If this fails for any reason it throws an error which bubbles up to the top level.  As an experiment I tried deleting the `statestore` directory and taking away write access from the `~/.ceramic` directory and ran the daemon.  I got an error: 
`Error: EACCES: permission denied, mkdir '/home/spencer/.ceramic/statestore/testnet-clay'`.  That kinda seems good enough for now?